doc/source/tutorial/optimize.rst

```diff
@@ -337,11 +337,73 @@ Hessian product example:
     >>> res.x
     array([1., 1., 1., 1., 1.])
 
+Trust-Region Truncated Generalized Lanczos / Conjugate Gradient Algorithm (``method='trust-krylov'``)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
The tutorial is generally well written and is clear! I left some thoughts for you to consider.